src/rules/no-unused-modules.js

```diff
@@ -12,46 +12,58 @@ import { dirname, join } from 'path';
 import readPkgUp from 'read-pkg-up';
 import values from 'object.values';
 import includes from 'array-includes';
+import tryCatch from 'try-catch';
```

Comment:
no thanks on adding this dep; i find deeply nested try-catch syntax highly superior to an arbitrary abstraction, even if it were a hundred levels deep.